src/Person/Person.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
is you pass a single object as a prop (person), you would not need to disable linter here